### PR TITLE
Updated spotting scope model.

### PR DESCRIPTION
**Pull request by p1nga**
I wanted to do a different model for the non deployed item.  In the future i will probably add a case or bag instead of it just being a copy of the actual deployed scope.

### CODE REVIEW

File: addons/spottingscope/data/model.cfg [hunk 1]
```diff
@@ -9,8 +9,12 @@ class CfgSkeletons
...
         skeletonInherit = "Default";
         skeletonBones[] = {
-            "otocvez","",
-            "otochlaven","otocvez"
+            "main_turret","",
+            "main_gun","main_turret",
+			
+			"leg_01","",
+			"leg_02","",
+			"leg_03",""
```
Comment (commy2) [inline addons/spottingscope/data/model.cfg]: tabs

File: addons/spottingscope/data/model.cfg [hunk 2]
```diff
@@ -37,13 +41,31 @@ class CfgModels {
...
+				angle1="rad +55";
+			};
+			class leg_02: leg_01 {
+				selection = "leg_02";
+				axis="leg_02_axis";
+			};
+			class leg_03: leg_01 {
+				selection = "leg_03";
+				axis="leg_03_axis";
+			};
```
Comment (commy2) [inline addons/spottingscope/data/model.cfg]: tabs

File: addons/spottingscope/CfgVehicles.hpp [hunk 3]
```diff
@@ -100,9 +100,10 @@ class CfgVehicles {
                     initAngleY = 0;
                     minAngleY = -100;
                     maxAngleY = 100;
-                    initFov = 0.014812;
-                    minFov = 0.014812;
-                    maxFov = 0.014812;
+					
+                    minFov = 0.0025;
+	                maxFov = 0.05;				
+					initFov= 0.05;
```
Comment (commy2) [inline addons/spottingscope/CfgVehicles.hpp]: tabs

File: addons/spottingscope/CfgVehicles.hpp [hunk 4]
```diff
@@ -89,9 +89,9 @@ class CfgVehicles {
                     initAngleY = 0;
                     minAngleY = -100;
                     maxAngleY = 100;
-                    initFov = 0.7;
-                    minFov = 0.7;
-                    maxFov = 0.7;
+                    initFov = 0.75;
+                    minFov = 0.25;
+                    maxFov = 1.25;
```
Comment (commy2) [inline addons/spottingscope/CfgVehicles.hpp]: white space

**Comment by jokoho48** [comment]
@p1nga if you fix the config stuff what commy2 mark we can merge this

**Pull request merged**